shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.js

```diff
@@ -364,6 +437,12 @@ exports.serveCaseToIrsInteractor = async (
     .updateDocketNumberRecord({ applicationContext })
     .validate();
 
+  await addDocketEntryForNANE({
```

Comment:
Take it or leave it, but it might be clearer to add the conditional before calling this function

Reply:
Despite what Github says, this is a suggestion, not a change request

Reply:
Thank you. Looking into how to make those changes.